erpnext/manufacturing/doctype/work_order/work_order.py

```diff
@@ -402,7 +402,7 @@ def update_operation_status(self):
 			elif flt(d.completed_qty) <= max_allowed_qty_for_wo:
 				d.status = "Completed"
 			else:
-				frappe.throw(_("Completed Qty can not be greater than 'Qty to Manufacture'"))
+				frappe.throw(_("Overproduction percentage limit has been surpassed, please adjust completed qty or change over production percentages in Manufacturing Settings."))
```

Comment:
Can you also show the max_allowed_qty_for_wo in the Error?

Something along the lines of, 'Maximum Allowed Qty is max_allowed_qty_for_wo'